systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractUpgradeST.java

```diff
@@ -147,26 +149,25 @@ protected void makeSnapshots() {
     }
 
     @SuppressWarnings("CyclomaticComplexity")
-    protected void changeKafkaAndLogFormatVersion(JsonObject procedures, JsonObject testParameters, ExtensionContext extensionContext) throws IOException {
+    protected void changeKafkaAndLogFormatVersion(Map<String, String> procedures, VersionModificationData testParameters, ExtensionContext extensionContext) throws IOException {
```

Comment:
Can you add the procedures into the `VersionModificationData`? Then you will just pass the `VersionModificationData` and remove the `Map<String, String>` param

Reply:
Yes, i would also prefer this better.